panda/src/glstuff/glCgShaderContext_src.cxx

```diff
@@ -944,30 +944,32 @@ update_shader_vertex_arrays(ShaderContext *prev, bool force) {
           }
         } else {
 #ifdef SUPPORT_FIXED_FUNCTION
-          switch (p) {
-          case CA_unknown:
-            break;
-          case CA_vertex:
-            glDisableClientState(GL_VERTEX_ARRAY);
-            break;
-          case CA_normal:
-            glDisableClientState(GL_NORMAL_ARRAY);
-            break;
-          case CA_color:
-            glDisableClientState(GL_COLOR_ARRAY);
+          if(_glgsg->has_fixed_function_pipeline()) {
```

Comment:
Like before, this check can also go away, since it shouldn't get assigned to CA_anything in the core profile case anyway.